lambdas/service/app.py

```diff
@@ -914,6 +915,7 @@ def repository_search(entity_type: str, item_id: str):
         selects entities whose donor organism count falls within both
         ranges, i.e., is exactly 5.
     '''),
+                                   facets=', '.join(app.facets)),
```

Comment:
```suggestion
        ranges, i.e., is exactly 5.''' + f'''
        
        Filter-able facets: {", ".join(app.facets)}
    ''')
```

Reply:
Note the lack of a trailing '.' to mirror the facet listing on the `sort` parameter.

Reply:
> Note the lack of a trailing '.' to mirror the facet listing on the `sort` parameter.

Ah yes, good suggestion but note that the facet listing in sort parameter are wrapped in a `enum` type, making them a valid list of available suggestion. Contrary to our case where the listing is done inside the description field.

Reply:
I still think the consistency would be nice, but since we seem to disagree on this point I'll defer to Hannes. Also, I don't think it makes sense to thumbs-up my comment if you're not going to implement it.